test/specs/scenarios/crossDevice.js

```diff
@@ -0,0 +1,186 @@
+import { it } from '../../utils/mochaw'
+import { localhostUrl, testDeviceMobileNumber } from '../../utils/config'
+import { goToPassportUploadScreen, uploadFileAndClickConfirmButton } from './sharedFlows.js'
+
+export const crossDeviceScenarios = (driver, screens, lang) => {
+  const {
+    welcome,
+    documentSelector,
+    documentUpload,
+    crossDeviceClientSuccess,
+    crossDeviceIntro,
+    crossDeviceLink,
+    crossDeviceMobileNotificationSent,
+    crossDeviceMobileConnected,
+    crossDeviceSubmit,
+    verificationComplete,
+    common
+  } = screens
+
+  const copy = common.copy(lang)
+
+  describe(`CROSS DEVICE scenarios in ${lang}`, () => {
+
+    const goToCrossDeviceScreen = async () => {
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
+      crossDeviceIntro.continueButton.click()
+    }
+
+    const waitForAlertToAppearAndSendSms = async () => {
+      driver.sleep(1000)
+      driver.switchTo().alert().accept()
+      crossDeviceLink.clickOnSendLinkButton()
+      driver.sleep(2000)
+    }
+
+    describe('cross device sync intro screen', async () =>  {
+      it('should verify UI elements on the cross device intro screen', async () => {
+        driver.get(localhostUrl + `?language=${lang}`)
+        welcome.primaryBtn.click()
+        documentSelector.passportIcon.click()
+        documentUpload.crossDeviceIcon.click()
+        crossDeviceIntro.verifyTitle(copy)
+        crossDeviceIntro.verifyIcons(copy)
+        crossDeviceIntro.verifyMessages(copy)
+      })
+    })
+
+    describe('cross device sync screen', async () => {
+      it('should verify UI elements on the cross device sync screen', async () => {
+        driver.get(localhostUrl + `?language=${lang}`)
+        goToCrossDeviceScreen()
+        crossDeviceLink.verifyTitle(copy)
+        crossDeviceLink.verifySubtitle(copy)
+        crossDeviceLink.verifyNumberInputLabel(copy)
+        crossDeviceLink.verifyNumberInput()
+        crossDeviceLink.verifySendLinkBtn(copy)
+        crossDeviceLink.verifyCopyLinkInsteadLabel(copy)
+        crossDeviceLink.verifyCopyToClipboardBtn(copy)
+        crossDeviceLink.verifyCopyLinkTextContainer()
+        crossDeviceLink.verifyDivider()
+      })
+
+      it('should change the state of the copy to clipboard button after clicking', async () => {
+        driver.get(localhostUrl + `?language=${lang}`)
+        goToCrossDeviceScreen()
+        crossDeviceLink.copyToClipboardBtn.click()
+        crossDeviceLink.verifyCopyToClipboardBtnChangedState(copy)
+      })
+
+      it('should display error when number is not provided', async () => {
+        driver.get(localhostUrl + `?language=${lang}`)
+        goToCrossDeviceScreen()
+        crossDeviceLink.typeMobileNumber('123456789')
+        crossDeviceLink.clickOnSendLinkButton()
```

Comment:
Minor: these 2 lines could probably be in a helper function since they're repeated in other tests

Reply:
Better to handle that as part of another story as this is getting way larger than the initial scope

Reply:
Cool. It's a minor thing anyway.